test/net/sourceforge/kolmafia/request/RequestTestBase.java

```diff
@@ -32,4 +86,171 @@ protected void expectSuccess(GenericRequest spy, String responseText) {
         .when(spy)
         .externalExecute();
   }
+
+  /**
+   * Set {@link #factory} to generate the given response if the path matches every one of {@code
```

Comment:
nit: "Sets ..."

"This is a fragment—a noun phrase or verb phrase, not a complete sentence. It does not begin with `A {@code Foo} is a...`, or `This method returns...`, nor does it form a complete imperative sentence like `Save the record.`. However, the fragment is capitalized and punctuated as if it were a complete sentence."

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Ditto elsewhere.